ignite/services/plugin/plugin_test.go

```diff
@@ -114,9 +114,9 @@ func TestNewPlugin(t *testing.T) {
 			expectedPlugin: Plugin{
 				repoPath:   "github.com/ignite/plugin@develop",
 				cloneURL:   "https://github.com/ignite/plugin",
-				cloneDir:   ".ignite/plugins/github.com/ignite/plugin@develop",
+				cloneDir:   ".ignite/plugins/github.com/ignite/plugin-eba2cc2492",
 				reference:  "develop",
-				srcPath:    ".ignite/plugins/github.com/ignite/plugin@develop/plugin1",
+				srcPath:    ".ignite/plugins/github.com/ignite/plugin-eba2cc2492/plugin1",
```

Comment:
He didn't drop the branch support, only changed the source path so it's unique among the different versions.

That said, now that I see that test, I wonder if this issue was actually required :sweat_smile:, because @branch/tag/hash are actually enough to provide that unicity... 

I'm not able to see the difference between this md5 discriminant and the previous one with @branch/tag/hash. Is there any actually @jeronimoalbi ?

Reply:
Either way, let's make sure we keep test cases for all supported methods:
- branch
- tag
- hash

Reply:
> He didn't drop the branch support, only changed the source path so it's unique among the different versions.
> 
> That said, now that I see that test, I wonder if this issue was actually required 😅, because @branch/tag/hash are actually enough to provide that unicity...
> 
> I'm not able to see the difference between this md5 discriminant and the previous one with @branch/tag/hash. Is there any actually @jeronimoalbi ?

I wondered the same at first @tbruyelle but I implemented it to avoid issues with branch or tag names like for example `feat/some-feature` or `package/v1.0.0`.

I guess we could have a different solution it we want to make names explicit instead of using hashes.

Reply:
Yes like escaping potential slashes in tag or branch names. At least in that case, the path are readable.

Reply:
Changeset 7b1fa97